docs/faq.rst

```diff
@@ -361,3 +361,16 @@ time consuming operation is very tricky matter.
 If you need global compression -- write own custom middleware. Or
 enable compression in NGINX (you are deploying aiohttp behind reverse
 proxy, is not it).
+
+
+How to manage ClientSession inside web server?
+----------------------------------------------
+
+:class:`aiohttp.ClientSession` should be created once for the lifetime
+of the server in order to benefit from connection pooling.
+
+Session saves cookies internally. If you don't need cookies processing
+use :class:`aiohttp.helpers.DummyCookieJar`. If you need separate cookies
```

Comment:
`aiohttp.DummyCookieJar` and `aiohttp.TCPConnector` -- these names are exported at top package level.